pkg/utils/wait_utils.go

```diff
@@ -0,0 +1,37 @@
+package utils
```

Comment:
A `utils` package is big NO :) because it will soon become a kitchen sink of unrelated things. Feel free to create a  `pkg/resourcepoll/wait.go`

Reply:
@sbose78 makes sense

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/101/commits/dafb96619aa2a0fc10b3bc2e2637222a85cd673e